build/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "ftlabs-screens-viewer",
-  "version": "1.0.10",
+  "version": "1.0.13",
```

Comment:
They were released and I tested them and found bugs. So I deleted the tag to stop them being available.